src/pose_service/include/pose_service/pose_service.hpp

```diff
@@ -0,0 +1,95 @@
+/*Copyright 2023 Brookhaven National Laboratory
+BSD 3 Clause License. See LICENSE.txt for details.*/
+#pragma once
+
+#include <cv_bridge/cv_bridge.h>
+#include <tf2_ros/static_transform_broadcaster.h>
+#include <tf2_ros/transform_broadcaster.h>
+#include <tf2/LinearMath/Quaternion.h>
+#include <tf2/LinearMath/Matrix3x3.h>
+
+#include <iostream>
+#include <thread>
+#include <memory>
+#include <string>
+#include <map>
+#include <cmath>
+#include <fstream>
+#include <vector>
+
+#include <rclcpp/rclcpp.hpp>
+#include <opencv2/opencv.hpp>
+#include <opencv2/aruco.hpp>
+
+#include <image_transport/image_transport.hpp>
+#include <sensor_msgs/msg/image.hpp>
+#include <filters/median.hpp>
+#include <geometry_msgs/msg/transform_stamped.hpp>
+#include <geometry_msgs/msg/quaternion.hpp>
+
+class PoseService : public rclcpp::Node
```

Comment:
To avoid overloading the word `Service` in ROS, I would rename this. 
Perhaps something like: `ObjectTransformNode` or `Image2PoseNode`, since this is a Node that subscribes to an image and broadcasts the transform to an object.

This change would have to propagate tot eh .cpp file of course, and be updated in the file name as well. 

The top-level package name can be adjusted as needed.

Reply:
Top level package name changed to ArucoPose

Reply:
In addition this PR contains fixes
1.  when ur_moveit.launch.py is launched, it looks for a missing file. [Missing file added](https://github.com/maffettone/erobs/blob/f854da99822cf3c313288d2869269837054299a9/src/ur3e_hande_moveit_config/config/joint_limits.yaml) 
2. Two new ros nodes to [open](https://github.com/maffettone/erobs/blob/f854da99822cf3c313288d2869269837054299a9/src/gripper_service/src/gripper_open.cpp) and [close](https://github.com/maffettone/erobs/blob/f854da99822cf3c313288d2869269837054299a9/src/gripper_service/src/gripper_close.cpp) the gripper is added. 
3. new [robot calibration parameters](https://github.com/maffettone/erobs/blob/f854da99822cf3c313288d2869269837054299a9/src/ur3e_hande_robot_description/config/ur3e/default_kinematics.yaml) added
4. devcontainer base package changed from humble-dev to humble-full : https://github.com/maffettone/erobs/blob/f854da99822cf3c313288d2869269837054299a9/.devcontainer/Dockerfile#L1